<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="1"
            failures="0"
            tests="0">

  <testsuite name="02-find-and-log"
    errors="1" failures="0" hostname="" id="" package="02-find-and-log" skipped="1"
    tests="0" time="0.001000" timestamp="">
  

  
    <system-err>
       An error occurred while creating a new GeckoDriver session: Connection refused to localhost:4444. If the Webdriver/Selenium service is managed by Nightwatch, check if &#34;start_process&#34; is set to &#34;true&#34;.
    </system-err>
  

    
    <error message="An error occurred while creating a new GeckoDriver session: Connection refused to localhost:4444. If the Webdriver/Selenium service is managed by Nightwatch, check if &#34;start_process&#34; is set to &#34;true&#34;."><![CDATA[
      Error: An error occurred while creating a new GeckoDriver session: Connection refused to localhost:4444. If the Webdriver/Selenium service is managed by Nightwatch, check if &#34;start_process&#34; is set to &#34;true&#34;.
    at Timeout._onTimeout (/Users/davidko/Work/nightwatch-sandbox/node_modules/nightwatch/dist/transport/selenium-webdriver/httpclient.js:81:39)
    at listOnTimeout (node:internal/timers:564:17)
    at process.processTimers (node:internal/timers:507:7)
      ]]></error>  

  
    
    <testcase
      name="Demo test ecosia.org" classname="02-find-and-log">
      <skipped />
    </testcase>
    
  
  </testsuite>
</testsuites>
